docs/reference/analysis/tokenfilters/cjk-bigram-tokenfilter.asciidoc

```diff
@@ -1,18 +1,178 @@
 [[analysis-cjk-bigram-tokenfilter]]
-=== CJK Bigram Token Filter
+=== CJK bigram token filter
+++++
+<titleabbrev>CJK bigram</titleabbrev>
+++++
 
-The `cjk_bigram` token filter forms bigrams out of the CJK
-terms that are generated by the <<analysis-standard-tokenizer,`standard` tokenizer>>
-or the `icu_tokenizer` (see {plugins}/analysis-icu-tokenizer.html[`analysis-icu` plugin]).
+Forms https://en.wikipedia.org/wiki/Bigram[bigrams] out of the CJK (Chinese,
+Japanese, and Korean) terms generated by the
+<<analysis-standard-tokenizer,standard tokenizer>> or the
+{plugins}/analysis-icu-tokenizer.html[ICU tokenizer].
```

Comment:
Strictly speaking, it will form bigrams from the CJK tokens produced by any tokenizer, so I'm not sure we need to refer to standard and icu here?

Reply:
Thanks @romseygeek. I removed the standard and ICU reference with cecd9bcfde4.